config/liveintent.php

```diff
@@ -41,11 +41,6 @@
     |
     */
     'auth' => [
-
-        'li_token' => [
-            'public_key' => env('LI_TOKEN_PUBLIC_KEY')
-        ]
-
```

Comment:
it looks like we are removing this in favor of doing it here: https://github.com/LiveIntent/laravel-common/pull/1/files#diff-64af6efa8d129988758fd1a3f84d368ba5242c179edc2d1e5b2cdd26a92dbeb1R31-R33 

can you explain why the change?

Reply:
Yep same reasoning as mentioned in this comment: https://github.com/LiveIntent/laravel-common/pull/1/files#r824009819